tools/ci_build/build.py

```diff
@@ -909,7 +907,6 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                 "-DCMAKE_OSX_DEPLOYMENT_TARGET=" + args.apple_deploy_target,
                 # we do not need protoc binary for ios cross build
                 "-Dprotobuf_BUILD_PROTOC_BINARIES=OFF",
-                "-Donnxruntime_ENABLE_BITCODE=" + ("OFF" if args.apple_disable_bitcode else "ON"),
                 "-DCMAKE_TOOLCHAIN_FILE=" + (
```

Comment:
If we're removing the ability to set onnxruntime_ENABLE_BITCODE from build.py do we still need that cmake variable?

Reply:
I want to keep this change minimal for easy cherry picking, will remove these in another PR